### PR TITLE
[FOV-38] Specific notification for Swap-Purchase-Burn buttons 

### PR DESCRIPTION
**Pull request by zvolkan**
https://fortisoeconomia.atlassian.net/browse/FOV-38

### CODE REVIEW

**Comment by RusSatellite** [comment]
make svault optoin but now hide this button because this option is not completed

File: components/Notification/element.tsx [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+`
+
+const Title = styled.div`
+  font-size: 20px;
+  line-height: 32px;
+  font-weight: 800;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  color: ${props => (props.type ? '#38B000' : '#B01A00')};
```
Comment (SweetDreamEng) [inline components/Notification/element.tsx:35]:
```suggestion
  font-size: 20px;
  line-height: 32px;
  font-weight: 800;
  display: flex;
  justify-content: center;
  align-items: center;
  color: ${props => (props.type ? '#38B000' : '#B01A00')};
```

```
  display: flex;
  justify-content: center;
  align-items: center;
  font-size: 20px;
  font-weight: 800;
  line-height: 32px;
  color: ${props => (props.type ? '#38B000' : '#B01A00')};
```

File: components/Notification/context.ts [hunk 2]
```diff
@@ -0,0 +1,5 @@
+import { createContext } from 'react'
+
+const NotificationContext = createContext(null)
```
Comment (SweetDreamEng) [inline components/Notification/context.ts:3]: @RusSatellite Please add the description of why this context is empty.

File: components/Notification/element.tsx [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+  useEffect(() => {
+    if (id) {
+      const timer = setTimeout(() => {
+        remove(id)
+      }, 5000)
+
+      return () => {
+        clearTimeout(timer)
+      }
+    }
```
Comment (SweetDreamEng) [inline components/Notification/element.tsx:67]: @RusSatellite Please take a reference of other timers as I reverted the change.

File: components/Notification/index.tsx [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+import useData from './useData'
+
+const NotificationContainer = ({ children }) => {
+  // type is success or error. true is success, false is error
+  const { notifications, success, error, remove } = useData()
+
+  return (
+    <NotificationContext.Provider value={{ success, error }}>
+      {notifications.map(({ id, title, txHash, type }) => (
+        <Notification key={id} id={id} action={() => {}} title={title} txHash={txHash} type={type} remove={remove} />
```
Comment (SweetDreamEng) [inline components/Notification/index.tsx:13]: @RusSatellite why do we need action prop here as its blank function?

File: components/Notification/element.tsx [hunk 5]
```diff
@@ -0,0 +1,125 @@
...
+  justify-content: center;
+  position: fixed;
+  z-index: 9999;
+  right: ${props => !props.defaultChecked && '0px'};
+  top: 30px;
+  left: ${props => props.defaultChecked && '0px'};
+
+  border: 1px solid;
+  border-color: ${props => (props.type ? 'green' : 'red')};
+  min-width: 280px;
```
Comment (SweetDreamEng) [inline components/Notification/element.tsx:19]: @RusSatellite Please reorder the attributes.

**Comment by SweetDreamEng** [comment]
@RusSatellite 
1. Add Jira ticket number in the title.
2. Add Jira ticket link in the description.
3. Please reorder all css attributes.

File: components/Notification/element.tsx (hunk 1 above)
Comment (RusSatellite) [reply components/Notification/element.tsx]: fixed with correct css order

File: components/Notification/context.ts (hunk 2 above)
Comment (RusSatellite) [reply components/Notification/context.ts]: added initial value and comment for createContext

File: components/Notification/index.tsx (hunk 4 above)
Comment (RusSatellite) [reply components/Notification/index.tsx]: removed action for Notification component

**Review by SweetDreamEng — APPROVED** [review]
LGTM!

**Pull request merged**